addons/sourcemod/scripting/vsh/stocks.sp

```diff
@@ -611,6 +682,45 @@ stock int TF2_CreateLightEntity(float flRadius, int iColor[4], int iBrightness)
 	return iGlow;
 }
 
+stock void TF2_ShowAnnotation(int[] iClients, int iCount, int iTarget, char[] sMessage, float flDuration = 5.0, char[] sSound = SOUND_NULL)
+{
+	//Create an annotation and show it to a specified array of clients
+	Event event = CreateEvent("show_annotation");
+	event.SetInt("follow_entindex", iTarget);
+	event.SetFloat("lifetime", flDuration);
+	event.SetString("text", sMessage);
+	event.SetString("play_sound", sSound);	//If this is missing, it'll try to play a sound with an empty soundpath
+	
+	for (int i = 0; i < iCount; i++)
+		event.FireToClient(iClients[i]);
+	
+	delete event;
+}
+
+stock void TF2_ShowAnnotationToAll(int iTarget, char[] sMessage, float flDuration = 5.0, char[] sSound = SOUND_NULL)
+{
+	//Create an annotation and show it to everyone. event.Fire() closes the handle on its own, so we don't need to do that here
+	Event event = CreateEvent("show_annotation");
+	event.SetInt("follow_entindex", iTarget);
+	event.SetFloat("lifetime", flDuration);
+	event.SetString("text", sMessage);
+	event.SetString("play_sound", sSound);
+	event.Fire();
+}
+
+stock void TF2_ShowAnnotationToClient(int iClient, int iTarget, char[] sMessage, float flDuration = 5.0, char[] sSound = SOUND_NULL)
+{
+	//Create an annotation and show it to a single client
+	Event event = CreateEvent("show_annotation");
+	event.SetInt("follow_entindex", iTarget);
+	event.SetFloat("lifetime", flDuration);
+	event.SetString("text", sMessage);
+	event.SetString("play_sound", sSound);
+	event.FireToClient(iClient);
+	
+	delete event;
+}
```

Comment:
I thought `TF2_ShowAnnotationToAll` and `TF2_ShowAnnotationToClient` would just create an array and call `TF2_ShowAnnotation`, without needing to copy-paste all of those events. Similar to how `EmitGameSoundToAll` and `EmitGameSoundToClient` is done